### ISSUE TITLE
Add bulk processor

### ISSUE THREAD

**Issue by mawrkus** [mawrkus/jason-the-miner#16]
?

```json
{
  "bulk": {
    "csv-file": {
      "path": "demos/data/in/github-search-queries.csv",
      "csv": {
        "delimiter": ","
      }
    },
    "concurrency": 2
  },
  "load": {
    "http": {
      "baseURL": "https://github.com",
      "url": "/search?l={language}&o=desc&q={query}&s=stars&type=Repositories"
    }
  },
  "parse": {
    "html": {
      "query": "< text({query})",
      "repos": [{
        "_$": ".repo-list .repo-list-item",
        "name": "h3 > a",
        "description": "div:first-child > p | trim",
        "stats": {
          "stars": ".col-2.text-right > a | trim",
          "last-update": "relative-time < attr(datetime)"
        }
      }]
    }
  },
  "transform": {
    "json-file": {
      "path": "demos/data/out/github-search-{language}.json"
    }
  }
}
```

github-search-queries.csv
```csv
language,query
JavaScript,scraper
Python,scraper
```

**Comment by mawrkus** [comment]
Implemented in #14

**Comment by mawrkus** [comment]
https://github.com/mawrkus/jason-the-miner/pull/14 merged
